tests/datasets_tests/molnet_tests/test_molnet.py

```diff
@@ -7,8 +7,9 @@
 from chainer_chemistry.datasets import NumpyTupleDataset
 from chainer_chemistry.datasets import molnet
 
-expect_bbbp_lengths = [1631, 203, 205]
+expect_bbbp_lengths = [1633, 203, 203]
```

Comment:
Do you know why this happen? Rdkit version??
If rdkit version issue, I want to fix recommended rdkit version for each chainer chemistry version.

Reply:
It seems that this change is caused by the difference between scaffold split and random split. But unexpected.

Reply:
I think this is not "unexpected" since you implement that StratifiedSplitter to have much "train" data for the remainder, right?